lhotse/cut.py

```diff
@@ -2774,11 +2774,18 @@ def load_audio(self, mixed: bool = True) -> Optional[np.ndarray]:
             reference_audio = reference_cut.load_audio()
             reference_energy = audio_energy(reference_audio)
 
-        mixer = AudioMixer(
-            self.tracks[0].cut.load_audio(),
-            sampling_rate=self.tracks[0].cut.sampling_rate,
-            reference_energy=reference_energy,
-        )
+        try:
+            mixer = AudioMixer(
+                self.tracks[0].cut.load_audio(),
+                sampling_rate=self.tracks[0].cut.sampling_rate,
+                reference_energy=reference_energy,
+            )
+        except NonPositiveEnergyError as e:
+            logging.warning(
+                f'{e} Cannot mix audios to the reference audio "{self.tracks[0].cut.id}"'
```

Comment:
I see, so in this case you'll return all zeroes. I guess it's as good solution as any -- can you add `"Returning zero signal."` at the end of the warning message?